tests/align/align_create_tensor_ff.py

```diff
@@ -0,0 +1,681 @@
+    elif operator_name == 'conv2d':
+        input_tensors, label, output_tensor = create_tensors_for_conv2d_ff(ffmodel)
+    elif operator_name == 'cos':
+        input_tensors, label, output_tensor = create_tensors_for_cos_ff(ffmodel)
+    elif operator_name == 'embedding':
+        input_tensors, label, output_tensor = create_tensors_for_embedding_ff(ffmodel)
+    elif operator_name == 'exp':
+        input_tensors, label, output_tensor = create_tensors_for_exp_ff(ffmodel)
+    elif operator_name == 'flat':
+        input_tensors, label, output_tensor = create_tensors_for_flat_ff(ffmodel)
+    elif operator_name == 'getitem':
+        input_tensors, label, output_tensor = create_tensors_for_getitem_ff(ffmodel)
+    elif operator_name == 'identity':
+        input_tensors, label, output_tensor = create_tensors_for_identity_ff(ffmodel)
+    elif operator_name == 'multiply':
+        input_tensors, label, output_tensor = create_tensors_for_multiply_ff(ffmodel)
+    elif operator_name == 'pool2d':
+        input_tensors, label, output_tensor = create_tensors_for_pool2d_ff(ffmodel)
+    elif operator_name == 'reducesum':
+        input_tensors, label, output_tensor = create_tensors_for_reducesum_ff(ffmodel)
+    elif operator_name == 'relu':
+        input_tensors, label, output_tensor = create_tensors_for_relu_ff(ffmodel)
+    elif operator_name == 'reshape':
+        input_tensors, label, output_tensor = create_tensors_for_reshape_ff(ffmodel)
+    elif operator_name == 'scalar_add':
+        input_tensors, label, output_tensor = create_tensors_for_scalar_add_ff(ffmodel)
+    elif operator_name == 'scalar_multiply':
+        input_tensors, label, output_tensor = create_tensors_for_scalar_multiply_ff(ffmodel)
+    elif operator_name == 'scalar_sub':
+        input_tensors, label, output_tensor = create_tensors_for_scalar_sub_ff(ffmodel)
+    elif operator_name == 'scalar_truediv':
+        input_tensors, label, output_tensor = create_tensors_for_scalar_truediv_ff(ffmodel)
+    elif operator_name == 'sigmoid':
+        input_tensors, label, output_tensor = create_tensors_for_sigmoid_ff(ffmodel)
+    elif operator_name == 'sin':
+        input_tensors, label, output_tensor = create_tensors_for_sin_ff(ffmodel)
+    elif operator_name == 'subtract':
+        input_tensors, label, output_tensor = create_tensors_for_subtract_ff(ffmodel)
+    elif operator_name == 'tanh':
+        input_tensors, label, output_tensor = create_tensors_for_tanh_ff(ffmodel)
+    elif operator_name == 'transpose':
+        input_tensors, label, output_tensor = create_tensors_for_transpose_ff(ffmodel)
+    elif operator_name == 'view_embedding':
+        input_tensors, label, output_tensor = create_tensors_for_view_embedding_ff(ffmodel)
+    else:
+        raise ValueError('Not include such Operator in Aligment Test', operator_name)
+        
+    compile_ffmodel(ffmodel)
+    dls = init_ffmodel(ffmodel, input_tensors, label)
```

Comment:
What does `dls` stand for? Can you write out the variable name unless it's really obvious and I'm just completely blanking :slightly_smiling_face:

Reply:
Thanks for the comment, changed name.